src/components/LandingPage/index.tsx

```diff
@@ -1,45 +1,39 @@
 'use client'
-import { Button } from '@nextui-org/react'
-import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+
+import { LoginCard } from '../LoginCard'
 
 const LandingPage = () => {
-  const router = useRouter()
+  const [type, setType] = useState('login')
 
   return (
-    <div className="bg-black h-screen flex justify-center items-center">
+    <div className="bg-black h-screen flex justify-center items-center relative">
       <video
         autoPlay
         muted
         loop
         className="absolute w-full h-full object-cover opacity-60"
       >
         <source
-
-          // We still need to find a better video here later on
           src="/videos/background.mp4"
           type="video/mp4"
         />
         Your browser does not support the video tag.
       </video>
 
       <div className="flex flex-col text-center text-white gap-5 drop-shadow-lg">
-        <h1 className="~text-5xl/8xl font-bold">Welcome to Anitube</h1>
-        <div className="~text-lg/2xl mx-auto w-4/5 md:w-full">
+        <h1 className="text-5xl font-bold">Welcome to Anitube</h1>
+        <div className="text-lg mx-auto w-4/5 md:w-full">
           <p>
             A lesson without pain is meaningless. That’s because you cannot gain
             something without sacrificing something else in return.
           </p>
           <p>- Welcome to the world of anime</p>
         </div>
-        <Button
-          color="secondary"
-          className="px-10 py-6 w-fit mx-auto text-xl"
-          onClick={() => {
-            router.push('/login')
-          }}
-        >
-          Login
-        </Button>
+        <LoginCard
+          type={type}
+          setType={setType}
+        />
```

Comment:
it doesn't handle only login so change its name to AuthCard

Reply:
fixed [b036224](https://github.com/dark-orga/anitube-website/pull/2/commits/b036224f01af39731e0a9ae8d060ce61d3b13dd9)